scripts/externalTests/runners/base.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+# ------------------------------------------------------------------------------
+# This file is part of solidity.
+#
+# solidity is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# solidity is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with solidity.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2023 solidity contributors.
+# ------------------------------------------------------------------------------
+
+import os
+import subprocess
+from abc import abstractmethod, ABCMeta
+from dataclasses import dataclass, field
+from pathlib import Path
+from shutil import rmtree
+from tempfile import mkdtemp
+from textwrap import dedent
+from typing import List, Set
+
+from test_helpers import (
+    download_project,
+    get_solc_short_version,
+    parse_command_line,
+    parse_custom_presets,
+    parse_solc_version,
+    replace_version_pragmas,
+    settings_from_preset,
+    SettingsPreset
+)
```

Comment:
What do you think about this style?
```suggestion
from test_helpers import download_project
from test_helpers import get_solc_short_version
from test_helpers import parse_command_line
from test_helpers import parse_custom_presets
from test_helpers import parse_solc_version
from test_helpers import replace_version_pragmas
from test_helpers import settings_from_preset
from test_helpers import SettingsPreset
```

This is how I usually do in my personal projects. The nice thing about it is that it's consistent when you have only 2 or 3 things to import - you don't have to suddenly switch to the parenthesized form.

Reply:
I'm fine with either ;)

Reply:
Ok, so let's go with this style for all imports. I.e. this as well:

```python
from dataclasses import dataclass
from dataclasses import field
```

and a few others.